src/Containers/Collection.php

```diff
@@ -47,10 +47,12 @@ public function load($value = null, $file = null)
     {
         $this->children = [];
 
-        if ($value) {
-            foreach ($value as $k => $v) {
-                $this->pushLoad($v, isset($file[$k]) ? $file[$k] : null);
-            }
+        if (empty($value)) {
+            return $this;
+        }
+
+        foreach ($value as $k => $v) {
+            $this->pushLoad($v, isset($file[$k]) ? $file[$k] : null);
         }
```

Comment:
I don't think a function with two returns is more clear than with one. I can see this in big functions but here there is only one "if".